src/ballet/sbpf/fd_sbpf_loader.c

```diff
@@ -366,7 +368,11 @@ fd_sbpf_load_shdrs( fd_sbpf_elf_info_t *  info,
 
       ulong sh_actual_size = shdr_get_loaded_size( &shdr[ i ] );
       ulong sh_virtual_end = sh_addr + sh_actual_size;
-      REQUIRE( sh_addr        == sh_offset               );
+
+      /* https://github.com/solana-labs/rbpf/blob/v0.8.0/src/elf.rs#L426 */
+      if ( elf_deploy_checks ){
```

Comment:
```suggestion
      if ( FD_UNLIKELY( elf_deploy_checks ) ){
```
Could you add these to all the deploy checks?